Guide-to-migrate-e2e-tests.md

```diff
@@ -0,0 +1,201 @@
+## Overview
+
+Currently, Oppia E2E tests are in a hybrid state where we have both WebdriverIO and Protractor test suites. The GSoC project **Migrate away from Protractor** aims to migrate all the existing E2E tests to WebdriverIO.
```

Comment:
Please spell out the acronym "E2E" for the first time you use it. Ditto elsewhere

Reply:
Done.